definitions/tools/bam_readcount.cwl

```diff
@@ -10,9 +10,7 @@ requirements:
     - class: DockerRequirement
       dockerPull: "mgibio/bam_readcount_helper-cwl:1.0.0"
 arguments: [
-    { valueFrom: $(runtime.outdir), position: -3 },
-    { valueFrom: " && ", shellQuote: false },
-    "/bin/cat", "$(runtime.outdir)/$(inputs.sample)_bam_readcount_snv.tsv", "$(runtime.outdir)/$(inputs.sample)_bam_readcount_indel.tsv"
```

Comment:
Looks like the `ShellCommandRequirement` can also be removed.

Reply:
Removed.